ext/AvizExt/viz/clustering.jl

```diff
@@ -78,31 +80,35 @@ Visualize clustered time series for each site and map.
 # Returns
 Figure
 """
-function ADRIA.viz.map(rs::Union{Domain,ResultSet}, data::AbstractArray,
-    clusters::Vector{Int64}; opts::Dict=Dict(), fig_opts::Dict=Dict(),
-    axis_opts::Dict=Dict())
-
+function ADRIA.viz.map(
+    rs::Union{Domain,ResultSet},
+    data::AbstractArray,

```

Comment:
I guess I missed this:

```suggestion
    data::AbstractArray{<:Real},

```

Providing at least some indication of the expected data types helps the compiler and reduces precompilation time.